client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -194,6 +189,8 @@ public void onUnregistrationFailed(NsdServiceInfo serviceInfo, int errorCode) {
                     }
                 });
         isAdvertising = true;
```

Comment:
I wonder if `isAdvertising` and `isDiscovering` should be true when there is no network available?